go.mod

```diff
@@ -1,57 +1,53 @@
 module github.com/hashicorp/hcp-scada-provider
 
-go 1.18
+go 1.23.0
+
+toolchain go1.24.0
 
 require (
-	github.com/hashicorp/go-hclog v1.2.1
-	github.com/hashicorp/hcp-sdk-go v0.23.0
-	github.com/hashicorp/net-rpc-msgpackrpc/v2 v2.0.0
-	github.com/hashicorp/yamux v0.0.0-20211028200310-0bc27b27de87
```

Comment:
can you exclude yamux? it is a little bit spooky to upgrade it. iirc it should also remove the upgrade to msgpackrpc which is desirable.

Reply:
msgpackrpc is probably fine since it's only a patch version update, but we could check the changelogs and see. I agree with not upgrading Yamux unless necessary.

Reply:
Rolled back the yamux version update.